### PR TITLE
changed 2019->2020

### PR DESCRIPTION
**Pull request by nlok5923**
As requested replaced all 2019 with 2020

fixes  : #18

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -33,7 +33,7 @@
 # <h3>Venue:</h3>
 Sabar Hostel premises
 # <h3>Time:</h3>
-Date: 5th October 2019 (Saturday) 10:00 AM to 6th October 2019 (Sunday) 10:00 AM.
+Date: 5th October 2029 (Saturday) 10:00 AM to 6th October 2020 (Sunday) 10:00 AM.
```
Comment (aashutoshrathi) [inline about.md:36]: Weekdays are same?

File: app/client/views/about/about.html [hunk 2]
```diff
@@ -54,7 +54,7 @@ <h3>Venue</h3>
         <h3>Time</h3>
     </div>
     <br>
-    <div style="text-align: center;"><b> Date: 5th October 2019 (Saturday) 10:00 AM to 6th October 2019 (Sunday) 10:00
+    <div style="text-align: center;"><b> Date: 5th October 2020 (Saturday) 10:00 AM to 6th October 2020 (Sunday) 10:00
```
Comment (aashutoshrathi) [inline app/client/views/about/about.html:57]: Same doubt here.

File: app/client/views/admin/user/user.html [hunk 3]
```diff
@@ -110,10 +110,10 @@
           name="year"
           ng-model="selectedUser.profile.graduationYear">
           <option value="">Graduation Year</option>
-          <option value="2019">2019</option>
-          <option value="2020">2020</option>
-          <option value="2021">2021</option>
-          <option value="2022">2022</option>
+          <option value="2019">2020</option>
```
Comment (aashutoshrathi) [inline app/client/views/admin/user/user.html:113]: Update the values here

File: app/client/views/application/application.html [hunk 4]
```diff
@@ -51,10 +51,10 @@
             name="year"
             ng-model="user.profile.graduationYear">
             <option value="">Graduation Year</option>
-            <option value="2019">2019</option>
-            <option value="2020">2020</option>
-            <option value="2021">2021</option>
-            <option value="2022">2022</option>
+            <option value="2019">2020</option>
```
Comment (aashutoshrathi) [inline app/client/views/application/application.html:54]: Values are not updated

File: app/server/templates/email-link-action/text.hbs [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Hi! Thanks for signing up for HackIIITV 2019. To verify your email, follow this link: {{verifyUrl}}
+Hi! Thanks for signing up for HackIIITV 2020. To verify your email, follow this link: {{verifyUrl}}
 
 Thanks,
 HackIIITV Team
```
Comment (aashutoshrathi) [inline app/server/templates/email-link-action/text.hbs:4]: Newline preferred.

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]

File: about.md (hunk 1 above)
Comment (nlok5923) [reply about.md]: changed weekdays in the latest commit

File: app/client/views/about/about.html (hunk 2 above)
Comment (nlok5923) [reply app/client/views/about/about.html]: changed weekdays in the latest commit

File: app/client/views/admin/user/user.html (hunk 3 above)
Comment (nlok5923) [reply app/client/views/admin/user/user.html]: values updated

File: app/client/views/application/application.html (hunk 4 above)
Comment (nlok5923) [reply app/client/views/application/application.html]: values updated

File: app/server/templates/email-link-action/text.hbs (hunk 5 above)
Comment (nlok5923) [reply app/server/templates/email-link-action/text.hbs:5]: Newline inserted in the latest commit.
changes done as requested please review @aashutoshrathi sir.

**Comment by nlok5923** [comment]
@iiitv/project-maintainers sir please review this pull request.

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Can you check why diff shows complete files instead of just `change`

**Comment by nlok5923** [comment]
Sir, because previously in my vscode I had modified setting to format on save because of that after every edit the code gets formatted. To resolve the same issue I had removed those setting from my vscode and after that made changes to the code.

**Comment by nlok5923** [comment]
opened a new #24 pull request for the same issue please review it.

**Pull request closed without merging**